docs/resources/cci_namespace.md

```diff
@@ -0,0 +1,82 @@
+---
+subcategory: "Cloud Container Instance (CCI)"
+---
+
+# huaweicloud_cci_namespace
+
+Manages a CCI namespace resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "namespace_name" {}
+
+resource "huaweicloud_cci_namespace" "test" {
+  name               = var.namespace_name
+  flavor             = "gpu-accelerated"
+  pool_size          = 15
+  recycling_interval = 30
+  auto_expend        = true
+  container_network  = true
+  rbac_enable        = true
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the CCI namespace resource.
+  If omitted, the provider-level region will be used. Changing this will create a new CCI namespace resource.
+
+* `flavor` - (Required, String, ForceNew) Specifies the CCI namespace flavor.
```

Comment:
this field is called **type** in console

Reply:
The argument name has been updated.